README.md

```diff
@@ -35,27 +35,7 @@ const config = {
 
 <details><summary markdown="span"><strong>For LoopBack 3 users</strong></summary>
 
-Use the [Data source generator](http://loopback.io/doc/en/lb3/Data-source-generator.html) to add a MongoDB data source to your application.
-The generator will prompt for the database server hostname, port, and other settings
-required to connect to a MongoDB database. It will also run the `npm install` command above for you.
-
-The entry in the application's `/server/datasources.json` will look like this:
-
-```javascript
-"mydb": {
-  "host": "myserver",
-  "port": 27017,
-  "url":  "",
-  "database": "test",
-  "password": "mypassword",
-  "name": "mydb",
-  "user": "me",
-  "authSource" : "admin",
-  "connector": "mongodb"
-}
-```
-
-Edit `datasources.json` to add any other additional properties that you require.
+`loopback-connector-mongodb@6` is not compatible with LoopBack 3, `loopback-connector-mongodb@5.4.0` is the last supported version.
```

Comment:
When we release v7 in the future, we will have to remember to update this sentence. Let's find a way how to convey the information so that we don't have to update this line in the future.

Also note that we may publish `5.5.0` sometimes in the future if we need to backport a (security?) fix that's not a trivial patch.

Example content for inspiration:

> Starting from the version `6.0.0`, this connector is no longer compatible with LoopBack 3. Please use the latest `5.x` version in your LoopBack 3 applications.

Reply:
We get questions periodically about which version of LB, juggler and connector that work together. Perhaps we can mention the juggler version as well (better in a table form)?

Reply:
> We get questions periodically about which version of LB, juggler and connector that work together. Perhaps we can mention the juggler version as well (better in a table form)?

That would be awesome 👏🏻 

I am ok to leave it out of scope of this pull request though, if it helps to get the changes landed faster.

Reply:
AFAICT, the support matrix table introduced by d0bb62d is already listing supported LoopBack versions, so I think this comment has been resolved.